captum/optim/_param/image/transform.py

```diff
@@ -73,15 +73,21 @@ def i1i2i3_transform() -> torch.Tensor:
         ]
         return torch.Tensor(i1i2i3_matrix)
 
-    def __init__(self, transform_name: str = "klt") -> None:
+    def __init__(self, transform_matrix: Union[str, torch.Tensor] = "klt") -> None:
```

Comment:
The ToRGB transform can optionally be turned off if one does not wish to use color decorrelation (like in Lucid & Lucent). The `transform` variable (previously called `transform_matrix`) can the matrix created by the dataset KLT function located in `_utils/image/dataset`, or it can be from something else. If you don't specify a matrix tensor, then you can use the default KLT or i1i2i3 one.